vector/v.net.salesman/v.net.salesman.html

```diff
@@ -165,8 +165,3 @@ <h3>TURNS SUPPORT</h3>
 Implementation: Stepan Turek 
 Documentation: Lukas Bocan
 Mentor: Martin Landa
-
-<!--
-<p>
-<i>Last changed: $Date$</i>
--->
```

Comment:
Line 37 missing one empty line.

Line 144 unnecessary one empty line.

```html
<em><a href="d.path.html">d.path</a></em>,
<em><a href="v.net.html">v.net</a></em>,
<em><a href="v.net.alloc.html">v.net.alloc</a></em>,
<em><a href="v.net.iso.html">v.net.iso</a></em>,
<em><a href="v.net.path.html">v.net.path</a></em>,
<em><a href="v.net.steiner.html">v.net.steiner</a></em>
```

should be 

```html
<em>
  <a href="d.path.html">d.path</a>,
  <a href="v.net.html">v.net</a>,
  <a href="v.net.alloc.html">v.net.alloc</a>,
  <a href="v.net.iso.html">v.net.iso</a>,
  <a href="v.net.path.html">v.net.path</a>,
  <a href="v.net.steiner.html">v.net.steiner</a>
</em>
```